.github/workflows/pkgci_test_pjrt.yml

```
@@ -32,9 +32,10 @@ jobs:
             pjrt_platform: cpu
           - runner: ubuntu-24.04
             pjrt_platform: cuda
-          # TODO: enable these AMD runners
-          # - runner: nodai-amdgpu-w7900-x86-64
-          #   pjrt_platform: rocm
+          - runner: nodai-amdgpu-w7900-x86-64
```

I think we'll want to change this to `iree-w7900` now. @kuhar can you confirm/help with that?
```suggestion
          - runner: iree-w7900
```

---

This should be `[Linux, X64, gfx1100]` if it doesn't have to run on w7900 specifically

---

Thank you both for look at it, I fixed the runner.